frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersionArchive.cy.ts

```diff
@@ -272,6 +281,63 @@ describe('Archiving version', () => {
     });
   });
 
+  it('Non archived version details page does have deployment tab', () => {
```

Comment:
nit: can we add "has" instead of "does have"? Also "Deployments" instead of "deployment".